tutorials/abandoned-browse-ccdp/data-modeling.md

```diff
@@ -0,0 +1,84 @@
+---
+position: 3
+title: Data Modeling
+---
+## Sending Events and Verifying Data
+
+After implementing the tracking setup, send some events and test that they are arriving in your data warehouse. If you don't have a data warehouse, you can sign up for a free trial of Snowflake.
+
+Once you have sent data to Snowflake, you can run the following query to verify the events are coming through:
+
+```sql
+SELECT 
+    domain_userid, 
+    user_id,
+    page_urlpath AS product_id, 
+    event_name,
+    page_title,
+    contexts_com_snowplowanalytics_snowplow_ecommerce_product_1,
+    contexts_com_snowplowanalytics_snowplow_web_page_1
+FROM DATABASE.ATOMIC.EVENTS
+WHERE 
+    DATE(load_tstamp) = CURRENT_DATE()
+ORDER BY load_tstamp DESC;
+```
+
+## Identifying Most Viewed but Not Added-to-Cart Products
+
+Once events are confirmed, use the following query to check that we can aggregate the data correctly. This query will be used in the next step to build an abandoned browse audience in Census.
+
+The SQL query identifies users who have viewed product pages today but have not completed a purchase. It aggregates data from Snowplow events to calculate the total time a user has engaged with product pages, checks if they have added items to their cart, and determines if they have previously received a win-back email. The second part of the query then selects the most engaged product for each user, and ensures that only users who are logged in are included.
```

Comment:
can this explanation be moved/copied into the sql directly please? easier to see what bit does what if directly labelled. might need a bit of editing to make sense as comments I dunno